qiskit/transpiler/passes/optimization/crosstalk_adaptive_schedule.py

```diff
@@ -23,8 +23,8 @@
 with simultaneous two-qubit and one-qubit gates. The method ignores
 crosstalk between pairs of single qubit gates.
 
-The method assumes that all qubits get measured simultaneously whether or not
-they need a measurement. This assumption is based on current device properties
+The method assumes that all qubits get measured simultaneously, whether
+they need a measurement or not. This assumption is based on current device properties
```

Comment:
The "or not" placement is technically valid in either place, but it's fine to pick whichever you think is more legible.